src/screens/Resources/ResourcesComponents/SingleItemList.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { View, TouchableOpacity, StyleSheet } from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { Text } from 'react-native-paper';
+import Collapsible from 'react-native-collapsible';
+import PropTypes from 'prop-types';
+import { nav } from '../../../constants';
+import { theme } from '../../../style';
+
+export default function SingleItemList({ snapshot, navigation, setLoading }) {
+  const data = snapshot.docs[0].data();
+  const { id } = snapshot.docs[0];
+  const [singleActive, setSingleActive] = useState(false);
+
+  return (
+    <View>
+      <TouchableOpacity
+        onPress={() => setSingleActive(!singleActive)}
+        style={SingleItemStyles.header}
+      >
+        <View style={SingleItemStyles.headerText}>
+          <Text style={SingleItemStyles.resourceText}>
+            {data.title}
+          </Text>
+        </View>
+        <View style={SingleItemStyles.headerIcon}>
+          <Icon name={singleActive ? 'chevron-up' : 'chevron-down'} />
+        </View>
+      </TouchableOpacity>
+      <Collapsible collapsed={!singleActive} style={SingleItemStyles.content}>
+        <Text>
+          {data.description}
+        </Text>
+
+        <View style={SingleItemStyles.moreButton}>
+          <Text
+            style={SingleItemStyles.resourceText}
+            onPress={() => {
+              setLoading(true);
+              navigation.push(nav.item, { resourceID: id });
+            }}
+          >
+            More&nbsp;
+            <Icon name="chevron-right" />
+          </Text>
+        </View>
+      </Collapsible>
+    </View>
+  );
+}
+
+const SingleItemStyles = StyleSheet.create({
```

Comment:
I'm not sure exactly what causes this problem, but the left/right margins for each list item are different when the accordion is extended vs not. (at least on iOS simulator, transportation -> big red bus) Is this intended?

Reply:
interesting but i cant test since i dont have an ios environment :(

Reply:
Ok, I don't think this is too big a deal, I can probably take another look at it after we merge this pr.